ui/app/components/app/transaction-list-item/index.scss

```diff
@@ -14,22 +14,24 @@
     padding: 16px 20px;
     display: grid;
     grid-template-columns: 45px 1fr 1fr 1fr;
-    grid-template-areas:
-      "identicon action status primary-amount"
-      "identicon nonce status secondary-amount";
+    grid-template-areas: "identicon action-and-nonce status amounts";
 
     @media screen and (max-width: $break-small) {
       padding: .5rem 1rem;
-      grid-template-columns: 45px 5fr 3fr;
-      grid-template-areas:
-        "nonce nonce nonce"
-        "identicon action primary-amount"
-        "identicon status secondary-amount";
+      grid-template-columns: 45px 3fr 3fr;
```

Comment:
This change results in the status being offset differently on certain transactions. e.g.:
![offset](https://user-images.githubusercontent.com/2459287/67048061-e5911100-f109-11e9-8f29-b73237577c89.png)

Something like this might be better: 
```suggestion
      grid-template-columns: 45px 3fr 1fr 1fr;
```
The important thing is that the second column gets a larger share than the last two, as it tends to be larger. I _think_ that should prevent the horizontal offset issue pictured, or at least it did locally while testing this.